src/yunohost/domain.py

```diff
@@ -301,3 +306,12 @@ def get_public_ip(protocol=4):
         logger.debug('cannot retrieve public IPv%d' % protocol, exc_info=1)
         raise MoulinetteError(errno.ENETUNREACH,
                               m18n.n('no_internet_connection'))
+
+def _get_maindomain():
+    with open('/etc/yunohost/current_host', 'r') as f:
+        maindomain = f.readline().rstrip()
```

Comment:
No idea, I just used the previous code and wrapped it in a function for it to be semantic. We can move to `strip()` if you want, but that shouldn't make any difference (except maybe if the user edited the file by hand) ?